lib/jsonapi/serializer.ex

```diff
@@ -76,9 +76,8 @@ defmodule JSONAPI.Serializer do
   end
 
   @spec encode_relationships(Plug.Conn.t(), serialized_doc(), tuple(), list()) :: tuple()
-  def encode_relationships(conn, doc, {view, data, _, _} = view_info, options) do
+  def encode_relationships(conn, doc, {view, _data, _, _} = view_info, options) do
     view.relationships()
-    |> Enum.filter(&data_loaded?(Map.get(data, elem(&1, 0))))
```

Comment:
This is the source of the breaking change. I think we could make the new behaviour opt-in at the cost of greater complexity.

Reply:
So the change here is if no data, then the empty relationship is included?  Is there a test that shows the change?